### ISSUE TITLE
Docs forget my 'Full API' preference

### ISSUE THREAD

**Issue by timhaines** [meteor/meteor#3441]
I'm signed in to Meteor.com when I visit.  Each time I view the docs, I need to switch to Full API.   

It'd be nice if Meteor.com remembered my preference and just showed the Full API every time.

**Comment by stubailo** [comment]
This works for me on Chrome desktop, Chrome iOS, and Safari iOS. Can you send me a video repro or something?

**Comment by timhaines** [comment]
@stubailo I've emailed a video to you.

**Comment by stubailo** [comment]
Hmm, it looks like it only happens if you click the link on meteor.com. If you type docs.meteor.com into the address bar, it does remember. Very interesting.

**Comment by hwillson** [comment]
Hi guys - I created a pull request (#3818) to help with this. Here's a breakdown of the issue:

- The top menu "Docs" link on meteor.com points to https://docs.meteor.com (since the site is SSL enabled)
- When you first click on that link you're taken to https://docs.meteor.com, but then quickly switched over to the non-SSL based version of the site - http://docs.meteor.com/#/basic/
- Since https://docs.meteor.com and http://docs.meteor.com both have separate localStorage, when coming from meteor.com the https://docs.meteor.com localStorage is being checked first (in docs/client/links.js line 61)
- When it can't be found the navigate function is called to get to '#/basic/'. The interesting thing here is that the **https**://docs.meteor.com URL is then switched to **http**://docs.meteor.com, likely because the navigate function in docs/client/lib/navigate.js is replacing the URL with a new URL built using Meteor.absoluteUrl(), meaning it's using the ROOT_URL environment variable (which is http:// based).
- Once over on http://docs.meteor.com, switching between basic/full works properly since the http://docs.meteor.com localStorage is consulted properly
- Long story short, adjusting the Meteor.absoluteUrl call in navigate.js to use the secure option should fix this issue

Thanks!

**Comment by glasser** [comment]
@hwillson we should close this now that we've merged #3818, right?

**Comment by hwillson** [comment]
Yes, I think so - thanks!
